src/Tribe/Query.php

```diff
@@ -216,12 +216,15 @@ public static function pre_get_posts( $query ) {
 						case 'custom':
 							// if the eventDisplay is 'custom', all we're gonna do is make sure the start and end dates are formatted
 							$start_date = $query->get( 'start_date' );
+							$start_date = $query->get( 'start_date' );
 							if ( $start_date ) {
-								$query->set( 'start_date', date_i18n( Tribe__Date_Utils::DBDATETIMEFORMAT, strtotime( $start_date ) ) );
+								$start_date_string = is_object($start_date) ? $start_date->date : $start_date;
```

Comment:
Nice, thanks for the PR! Here and below, how about swapping out the `is_object()` call with an `instanceof` test instead (just to further tighten things up)?